spec/lib/saml/ssoe_user_spec.rb

```diff
@@ -93,10 +93,10 @@
 
       it 'has various important attributes' do
         expect(subject.to_hash).to eq(
-          email: 'vets.gov.user+262@example.com',
-          first_name: 'JERRY',
+          email: nil,
```

Comment:
Why are these nil? Is this expected?

Reply:
Yes, we're not getting an email back for IAL1 users. I'll update the PR with the responses for IAL1 & 2.

Reply:
Turns out this is just an issue of adding `email` as a requested attribute, then it will return for IAL1